contracts/src/v0.8/functions/dev/0_0_0/FunctionsBillingRegistry.sol

```diff
@@ -91,24 +95,28 @@ contract FunctionsBillingRegistry is
   error Reentrant();
 
   mapping(address => uint96) /* oracle node */ /* LINK balance */ private s_withdrawableTokens;
+
   struct Commitment {
-    uint64 subscriptionId;
-    address client;
-    uint32 gasLimit;
+    uint64 subscriptionId; // -┐
+    address client; //         |
+    uint32 gasLimit; // -------┘
+    address don; // -----------┐
+    uint96 donFee; // ---------┘
+    uint96 registryFee; // ----┐
+    uint96 estimatedCost; //   |
+    uint40 timestamp; // ------┘
```

Comment:
I highly encourage to use these struct packing comments for every struct so the person writing the code is forced to actively think about struct packing.